src/main/kotlin/specific/lotto/domain/Round.kt

```diff
@@ -1,9 +1,14 @@
 package specific.lotto.domain
 
 class Round(val player: Player, val winningNumbers: WinningNumbers) {
-    val winningResult = player.tickets
-        .map { Rank.decideRank(it.countSameNumber(winningNumbers), it.hasBonusNumber(winningNumbers)) }
-        .let { WinningResult(it) }
+    val winningResult = makeWinningResult()
+
+    private fun makeWinningResult(): WinningResult {
+        return player.tickets
+            .let { it.manualTickets + it.autoTickets }
+            .map { Rank.decideRank(it.countSameNumber(winningNumbers), it.hasBonusNumber(winningNumbers)) }
+            .let { WinningResult(it) }
```

Comment:
이러한 구문을 함수 reference로 넘길 수 있습니다.
```suggestion
            .let(::WinningResult)
```
lambda로 작성하는 것과 어떤 차이가 있는지 참고할 수 있는 글을 소개해드릴게요 :slightly_smiling_face:
https://proandroiddev.com/kotlin-lambda-vs-method-reference-fdbd175f6845
참고로 저는 가능한 곳에서는 꼭 함수 reference를 활용하는 것을 지향합니다. lambda로 가져온 데이터를 별도로 조작하지 않고 그대로 함수 시그니처로 넘긴다는 것을 드러낼 수 있기 때문입니다.

Reply:
좋은 방법 알려주셔서 감사합니다. 해당 부분은 수정했고 추후 미션 진행 시에도 적극 활용하도록 하겠습니다.